coremark_pro/test_coremark_pro.yaml

```diff
@@ -0,0 +1,50 @@
+test:
+    type: harness
+    harness: coremark_pro.py
+    nightly: true
+
+    targets:
+      value: [armv7a7-imx6ull-evk, armv7a9-zynq7000-qemu, armv7a9-zynq7000-zedboard, armv7m4-stm32l4x6-nucleo, armv7m7-imxrt106x-evk, armv7m7-imxrt117x-evk, ia32-generic-qemu, riscv64-generic-qemu]
+
+    tests:
+      - name: cjpeg-rose7-preset
+        targets:
+          exclude: [armv7m4-stm32l4x6-nucleo]
+        execute: /usr/bin/coremark-pro/cjpeg-rose7-preset -v0
+
+      - name: core
+        execute: /usr/bin/coremark-pro/core -v0
+
+      - name: linear_alg-mid-100x100-sp
+        execute: /usr/bin/coremark-pro/linear_alg-mid-100x100-sp -v0
```

Comment:
Maybe binaries should be installed in `/bin` as in other ports so then we can use only test name.